### DISCUSSION TITLE
Implement "Logout all devices" feature

### DISCUSSION THREAD

**Discussion by WhamboMPS** [simple-login/app#583] [Feature Requests]
Implement a feature that forces all instances of SimpleLogin to require active login (re-login) before they can be used again.. This would include web pages, browser plugins, and most especially mobile devices. The use case is for a lost or stolen device (or laptop).

**Comment by nguyenkims** [comment]
@WhamboMPS Deleting all API keys will log out on all devices. This is indeed not the best way so we'll add a more convenient button to remove all API Keys at once :).

**Reply by IkelAtomig** [reply]
Removing the API keys will only remove the sessions in the apps and services it is used not the Logged in Web sessions in app.simplelogin.io
